static/app/views/settings/components/forms/richListField.tsx

```diff
@@ -249,7 +259,38 @@ const ItemList = styled('ul')`
   padding: 0;
 `;
 
-const Item = styled('li')<{disabled?: boolean}>`
+const getItemStyle = (theme: Theme, disabled: boolean, error: boolean) => {
+  if (disabled) {
+    return `
+      opacity: 0.65;
+      cursor: not-allowed;
+      background-color: ${theme.button.default.background};
+      border: 1px solid ${theme.button.default.border};
+      color: ${theme.button.default.color};
+    `;
+  }
+
+  if (error) {
+    return `
+      color: ${theme.black};
+      border: 1px solid ${theme.red300};
+      &,
+      button,
+      button:hover,
+      button:focus {
+        background-color: ${theme.red100};
```

Comment:
Should these pull from `theme.button.danger` or `theme.alert.error`?

Reply:
I've updated this component and removed this style